### DISCUSSION TITLE
Plugin barcode-scanning and Capacitor 6 support

### DISCUSSION THREAD

**Discussion by tibetoine** [capawesome-team/capacitor-mlkit#177] [Q&A] [answered]
Hello,

I hope you're doing well.

I wanted to clarify if the barcode-scanning plugin is compatible with Capacitor V6. The [breaking changes file](https://github.com/capawesome-team/capacitor-mlkit/blob/main/packages/barcode-scanning/BREAKING.md#version-6xx) mentions that "This plugin now supports Capacitor 5 only." However, since there is a section dedicated to Capacitor V6, I'm a bit unsure about its compatibility.

Could you please confirm this?

Thank you!

**Comment by robingenz** [comment, marked as answer]
Yes, the latest version supports Capacitor 6.

**Comment by tibetoine** [comment]
Thank you very much for your prompt and detailed response.
